### PR TITLE
Add archbuild man page

### PR DESCRIPTION
**Pull request by tsuibin**

### CODE REVIEW

**Comment by jelly** [comment]
Please also include a link to archbuild.1.asciidoc in https://github.com/archlinux/devtools/blob/master/doc/devtools.7.asciidoc

**Comment by felixonmars** [comment]
The `MANS` list in Makefile should be updated as well, I think.

**Review by felixonmars — APPROVED** [review]

File: doc/archbuild.1.asciidoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* kde-unstable-x86_64-build 
+* multilib-testing-build 
+* testing-x86_64-build
+* extra-x86_64-build
+* multilib-build 
+* gnome-unstable-x86_64-build  
+* multilib-staging-build     
+* staging-x86_64-build
+
+The symlink used to run it will be inspected by archbuild, to determine which target you want it to use.
```
Comment (jelly) [inline doc/archbuild.1.asciidoc:27]: Maybe this should also explain the configuration file loading when you use `staging-x86_64-build`
```
devtools /usr/share/devtools/pacman-kde-unstable.conf
```

File: doc/archbuild.1.asciidoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+archbuild - a script to build an Arch Linux package inside a clean chroot. 
+
+Synopsis
+--------
+archbuild [options] -- [makechrootpkg args]
+
+Description
+-----------
+
+'archbuild' is a script to build an Arch Linux package. devtools includes archbuild, but you don't run this command directly.
```
Comment (jelly) [inline doc/archbuild.1.asciidoc:15]:
```suggestion
'archbuild' is a script to build an Arch Linux package. devtools includes archbuild but should only be used via one of the included symlinks:
```

Comment (jelly) [reply doc/archbuild.1.asciidoc:15]: I would reword it to not say "you" or that you don't run it directly.

**Review by jelly — CHANGES_REQUESTED** [review]

**Comment by jelly** [comment]
Merged via fd17f53cef61ca5e25219fcc0d555e154cd0af59
